go.mod

```diff
@@ -0,0 +1,80 @@
+	github.com/ghodss/yaml v1.0.0
+	github.com/go-openapi/loads v0.17.2
+	github.com/go-openapi/runtime v0.17.2
+	github.com/go-openapi/spec v0.19.2
+	github.com/go-redis/cache v6.3.5+incompatible
+	github.com/go-redis/redis v6.15.1+incompatible
+	github.com/gobuffalo/packr v1.11.0
+	github.com/gobwas/glob v0.2.3
+	github.com/gogits/go-gogs-client v0.0.0-20190616193657-5a05380e4bc2
+	github.com/gogo/protobuf v1.1.1
+	github.com/golang/protobuf v1.2.0
+	github.com/google/go-jsonnet v0.10.0
+	github.com/google/shlex v0.0.0-20181106134648-c34317bd91bf
+	github.com/googleapis/gnostic v0.1.0 // indirect
+	github.com/grpc-ecosystem/go-grpc-middleware v0.0.0-20190222133341-cfaf5686ec79
+	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
+	github.com/grpc-ecosystem/grpc-gateway v1.3.1
+	github.com/improbable-eng/grpc-web v0.0.0-20181111100011-16092bd1d58a
+	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51
+	github.com/mitchellh/go-wordwrap v1.0.0 // indirect
+	github.com/patrickmn/go-cache v2.1.0+incompatible
+	github.com/pkg/errors v0.8.0
+	github.com/pquerna/cachecontrol v0.0.0-20180306154005-525d0eb5f91d // indirect
+	github.com/prometheus/client_golang v0.9.2
+	github.com/robfig/cron v1.2.0
+	github.com/rs/cors v1.6.0 // indirect
+	github.com/sirupsen/logrus v1.4.2
+	github.com/skratchdot/open-golang v0.0.0-20160302144031-75fb7ed4208c
+	github.com/soheilhy/cmux v0.1.4
+	github.com/spf13/cobra v0.0.4
+	github.com/spf13/pflag v1.0.3
+	github.com/stretchr/testify v1.3.0
+	github.com/vmihailenco/msgpack v3.3.1+incompatible
+	github.com/yudai/gojsondiff v1.0.1-0.20180504020246-0525c875b75c
+	github.com/yudai/golcs v0.0.0-20170316035057-ecda9a501e82 // indirect
+	github.com/yuin/gopher-lua v0.0.0-20190115140932-732aa6820ec4
+	golang.org/x/crypto v0.0.0-20190611184440-5c40567a22f8
+	golang.org/x/net v0.0.0-20190613194153-d28f0bde5980
+	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
+	golang.org/x/sync v0.0.0-20190423024810-112230192c58
+	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2 // indirect
+	gonum.org/v1/gonum v0.0.0-20190621125449-90b715451587 // indirect
+	google.golang.org/genproto v0.0.0-20180817151627-c66870c02cf8
+	google.golang.org/grpc v1.15.0
+	gopkg.in/go-playground/webhooks.v5 v5.11.0
+	gopkg.in/mgo.v2 v2.0.0-20160818020120-3f83fa500528 // indirect
+	gopkg.in/src-d/go-git.v4 v4.9.1
+	gopkg.in/yaml.v2 v2.2.2
+	k8s.io/api v0.0.0-20190816222004-e3a6b8045b0b
+	k8s.io/apiextensions-apiserver v0.0.0-20190404071145-7f7d2b94eca3
+	k8s.io/apimachinery v0.0.0-20190816221834-a9f1d8a9c101
+	k8s.io/client-go v11.0.1-0.20190816222228-6d55c1b1f1ca+incompatible
+	k8s.io/code-generator v0.0.0-20190711102700-42c1e9a4dc7a
+	k8s.io/gengo v0.0.0-20190327210449-e17681d19d3a // indirect
+	k8s.io/klog v0.3.3
+	k8s.io/kube-aggregator v0.0.0-20190711105720-e80910364765
+	k8s.io/kube-openapi v0.0.0-20190502190224-411b2483e503
+	k8s.io/kubernetes v1.15.0-alpha.0.0.20190914015840-8fca2ec50a61
```

Comment:
I saw that 1.15.7 (which is latest for 1.15) now has support for modules, so we can get rid of the pseudo versions